docs/workgroup/charter.rst

```diff
@@ -0,0 +1,136 @@
+===================================
+CPython Documentation Working Group
+===================================
+
+Purpose & Common Goals
+----------------------
+
+This workgroup will support efforts to improve
+and maintain CPython's documentation through curation,
+internationalization, and special projects. Its focus will also include
+tutorials, how-to documents, the devguide and other documentation in support
+of the PSF's mission to expand the global Python userbase.
+Quality and effectiveness will be key goals as we build a
```

Comment:
Present tense
```suggestion
Quality and effectiveness are key goals as we build a
```

Reply:
The simple future tense is used throughout the rest of the "Purpose & Common Goals" section. If we do want to change it, we should do so consistently throughout, not just in one arbitrary place.

Reply:
Yep, I think we should change it everywhere; I didn't want to spam with tense adjustments though! The review comment here ("present tense") is rather terser than I remembered, though, so sorry for that.

A

Reply:
Sorry, yeah, that was very unclear. 

Anyway, done.